src/components/connectors/card/index.tsx

```diff
@@ -53,22 +63,56 @@ function ConnectorCard({ cta, docsUrl, logo, title, details }: Props) {
-                                    alignItems: 'center',
-                                }}
+                        <Stack
+                            sx={{
+                                marginBottom: recommended ? 1 : 2,
+                            }}
+                        >
+                            <Box
+                                sx={
+                                    recommended
+                                        ? {
+                                              ...connectorImageBackgroundSx,
+                                              mb: 0,
+                                              borderBottomLeftRadius: 0,
+                                              borderBottomRightRadius: 0,
+                                          }
+                                        : connectorImageBackgroundSx
+                                }
                             >
-                                <Box>{logo}</Box>
+                                <Stack
+                                    sx={{
+                                        justifyContent: 'center',
+                                        alignItems: 'center',
+                                    }}
+                                >
+                                    <Box>{logo}</Box>
 
-                                {docsUrl ? (
-                                    <ExternalLink link={docsUrl}>
-                                        <FormattedMessage id="terms.documentation" />
-                                    </ExternalLink>
-                                ) : null}
-                            </Stack>
-                        </Box>
+                                    {docsUrl ? (
+                                        <ExternalLink link={docsUrl}>
+                                            <FormattedMessage id="terms.documentation" />
+                                        </ExternalLink>
+                                    ) : null}
+                                </Stack>
+                            </Box>
+
+                            {recommended ? (
+                                <Box
+                                    sx={{
+                                        bgcolor: teal[500],
+                                        color: 'white',
+                                        borderBottomLeftRadius:
+                                            connectorImageBackgroundRadius,
+                                        borderBottomRightRadius:
+                                            connectorImageBackgroundRadius,
+                                        opacity: 0.75,
+                                        textAlign: 'center',
+                                    }}
+                                >
+                                    <FormattedMessage id="common.recommended" />
+                                </Box>
+                            ) : null}
```

Comment:
Basic box that leaned on the approach of estuary.dev to follow. The purple color did not work well since the CTA is already using that. `teal[500]` with some opacity works well for both dark and light so went with that.